src/Xamarin.Android.Build.Tasks/Tasks/Proguard.cs

```diff
@@ -176,6 +181,33 @@ protected override string GenerateFullPathToTool ()
 		string ProguardHome {
 			get { return UseProguard ? ProguardToolPath : Path.GetDirectoryName (Path.GetDirectoryName (ProguardJarPath)); }
 		}
+
+		protected override bool HandleTaskExecutionErrors ()
+		{
+			if (foundError) {
+				errorText.Clear ();
+				foreach (var line in errorLines)
+					errorText.Append (line);
+				Log.LogCodedError ($"XA4307", $"{errorText}");
+				return !Log.HasLoggedErrors;
+			}
+			return base.HandleTaskExecutionErrors ();
+		}
+
+		protected override void LogEventsFromTextOutput (string singleLine, MessageImportance messageImportance)
+		{
+			if (singleLine.Contains ("proguard.ParseException")) {
+				Log.LogMessage (MessageImportance.High, singleLine);
```

Comment:
Should this "trim out" the `proguard.ParseException` bit?  Is that at the beginning of the line or anything?